<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;你你你&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>228</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>160</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>用户名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>170</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>240</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>240</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>密码   </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>841</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image:url(:/20210817161503802.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>-10</y>
      <width>641</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string>车南大学智慧银行管理系统</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonexit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>380</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image:rgb(135, 163, 255)</string>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonin">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>310</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image:rgb(170, 0, 0)</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>430</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>登录时需已安装 Windows 或 Mac “网上银行登录安全控件”</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>-50</x>
      <y>0</y>
      <width>881</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image:url(:/mainwindow.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_6</zorder>
   <zorder>label_3</zorder>
   <zorder>widget_4</zorder>
   <zorder>label</zorder>
   <zorder>lineEdit</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>label_2</zorder>
   <zorder>label_4</zorder>
   <zorder>pushButtonexit</zorder>
   <zorder>pushButtonin</zorder>
   <zorder>label_5</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>787</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menubanbenxinxi">
    <property name="title">
     <string>版本信息</string>
    </property>
    <addaction name="actionchk"/>
   </widget>
   <addaction name="menubanbenxinxi"/>
  </widget>
  <action name="actionchk">
   <property name="text">
    <string>查看</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
